### PR TITLE
fix: wait for the next tick

### PR DESCRIPTION
**Pull request by suits-at**
according to https://vue-test-utils.vuejs.org/guides/#using-nexttick we have to wait for the next tick for this test to pass

### CODE REVIEW

File: docs/guides/testing-single-file-components-with-mocha-webpack.md
Comment (JessicaSachs) [inline docs/guides/testing-single-file-components-with-mocha-webpack.md]: Thank you for the PR! Prettier is complaining about this semicolon. `yarn run format` will fix it.

**Review by JessicaSachs — CHANGES_REQUESTED** [review]
Thank you for finding and fixing this! There is a small formatting change needed before merging. You can exec prettier via `yarn run format`.

We might run the linter as a githook to avoid this problem in the future.

**Comment by dobromir-hristov** [comment]
I can add prettier to run auto on my commit lint branch.

File: docs/guides/testing-single-file-components-with-mocha-webpack.md
Comment (dobromir-hristov) [reply docs/guides/testing-single-file-components-with-mocha-webpack.md]: You can also use `wrapper.vm.$nextTick()`, but this could be a bit more future proof, as nextTick will no longer be avail on the vm in V3...

**Review by lmiller1990 — COMMENTED** [review]
LGTM once prettier is run

**Pull request merged**
